### ISSUE TITLE
scroll out of sync with different column heights

### ISSUE THREAD

**Issue by m-rnd** [nesyou01/LazyStaggeredGrid#7]
When the columns differ in height (sum of all items), they are not scrolling in sync anymore once the smallest column reaches the end of the screen. After that, the other column(s) scroll further with the small column staying at the screen edge. 
Expected behaviour: the smaller column scrolls together with the other columns with increasing distance to the screen edge (like with a padding).

Tested with Pixel 6, Android 12, StaggeredGrid v1.2.0

**Comment by Subfly** [comment]
yes, having the same issue. but looking into source code, it is pretty impossible to handle unless the column with smaller height fills the entire available height created by the longer one. I have spent hours on it but couldn't fix that :/
